pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -42,9 +42,10 @@ type MultiNodeConsolidation struct {
 }
 
 func NewMultiNodeConsolidation(consolidation consolidation) *MultiNodeConsolidation {
```

Comment:
```suggestion
func NewMultiNodeConsolidation(c consolidation) *MultiNodeConsolidation {
```

nit: Naming difference across constructors